### DISCUSSION TITLE
First Time w/ pfSense and Security Onion

### DISCUSSION THREAD

**Discussion by SighberNerd** [Security-Onion-Solutions/securityonion#8561] [Unsupported Versions] [answered]
Trying to figure out how to export pfSense Logs from my Netgate firewall, setup with VLANs and the various FW rules, into a Security Onion VM located on my MacBookPro M2 running Security Onion in VMWare.  I see tutorials for how to pipe to a log server but I am not running a server, I just want to review some of the logging from my firewall in Security Onion to see what it looks like; how well SO parses pfSense Syslog files.  Can anyone assist? I know #LMGTFY

**Comment by dougburks** [comment, marked as answer]
Please see the pfSense section of our documentation:
https://docs.securityonion.net/en/2.3/other-supported-logs.html#example-pfsense
